.github/ISSUE_TEMPLATE/simple_question.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Simple question
+about: Ask a question about `php-github-api`
+
+---
+
+**My question**
```

Comment:
Is this really needed? Why is the benefit of using this template than just leave a blank page?